### ISSUE TITLE
Don't warn about missing usage file unless usage syncing enabled

### ISSUE THREAD

**Issue by adamconnelly** [infracost/infracost#832]
If you run the `infracost breakdown` command and set the log-level to warn, a warning is output about the usage file not being specified, even if the `--sync-usage-file` flag hasn't been specified.

Here's an example:

```shell
$ infracost breakdown --path plan.json --format=json --log-level=warn > infracost.json 
time="2021-06-25T16:38:15+01:00" level=warning msg="Can't sync usage file as it's not specified"
```

I'd only expect this if the `--sync-usage-file` flag was specified, for example:

```shell
infracost breakdown --path spacelift.json --format=json --log-level=warn --sync-usage-file
```

This is with Infracost 0.9.1.

**Comment by adamconnelly** [comment]
@alikhajeh1 here's a PR if you want it: https://github.com/infracost/infracost/pull/833

I've left a bit of a question in the description because it looks to me like the warning could just be removed, but I didn't wanna take that step without checking first.

**Comment by alikhajeh1** [comment]
Thanks @adamconnelly! This was released in v0.9.2 today.
